frontstage/controllers/conversation_controller.py

```diff
@@ -72,10 +72,11 @@ def send_message(message_json):
     try:
         response = session.post(url, headers=headers, data=message_json)
         response.raise_for_status()
-    except HTTPError as ex:
+        logger.info("Message sent successfully")
```

Comment:
Could you put a ```python

Reply:
Line 64 in create_message.py gives us the message id and party id, so I think that should be fine 👍

Reply:
It's not good enough, lots of the time when you're debugging an issue and looking through logs, you filter on an ID to begin with, see what messages you get then go from there.  And we want to make that part as easy as possible whereever we can

Think about in SDX, pretty much all of the lines have a tx_id on them and when you search in splunk, it's really easy to see the whole flow.  If you want anything that hasn't got that id then it's more of a pain to find generally